wdl/transforms/draft3/src/main/scala/wdl/draft3/transforms/ast2wdlom/EnhancedDraft3Ast.scala

```diff
@@ -53,5 +57,18 @@ object EnhancedDraft3Ast {
         result <- asVector.traverse[ErrorOr, A](item => toA.run(item).toValidated).toEither
       } yield result
     }
+
+    /**
+      * Gets an attribute on this Ast as an Optional Ast, returns an empty Option if the attribute is empty.
+      */
+    def getAttributeAsOptionalAst[A](attr: String)(implicit toA: CheckedAtoB[AstNode, Option[A]]): Checked[Option[A]] =  {
```

Comment:
Looks like this should really be called `getAttributeAsOptional[A]` since you could put any arbitrary result type in there